tests/unit/test_api_es.py

```diff
@@ -909,28 +997,58 @@ def test_applyFilters_no_filters_date_ort(self, testInstance, mocker):
-            mocker.call('lang_parent', 'nested', path='editions.languages'),
-            mocker.call('languages', 'terms', field='editions.languages.language', size=200),
-            mocker.call('editions_per', 'reverse_nested'),
-            mocker.call('formats', 'terms', field='editions.formats', size=10),
-            mocker.call('editions_per', 'reverse_nested')
-        ])
+        assert testInstance.query.aggs.to_dict()['aggs']['govDoc'] == {'terms': {'field': 'is_government_document', 'size': 2}}
+        assert testInstance.query.aggs.to_dict()['aggs']['govDoc_filter'] == {'filter': {'term': {'is_government_document': True}}}
+
+    def test_applyAggregations_govDoc_and_other_aggregations(self, testInstance):
+
+        testInstance.query = Search()
+        testInstance.dateSort = None
+        testInstance.displayAggregation = None
+        testInstance.appliedAggregations = [A(
+                'filter', **{'range': {'editions.publication_date': 'testDate'}}
+            ), A(
+                'filter', **{'terms': {'editions.formats': 'testFormat'}}
+            ), A('filter', **{'exists': {'field': 'editions.formats'}})]
+        testInstance.govDocAgg = A('filter', **{'term': {'is_government_document': False}})
+
+        testInstance.applyAggregations()
+        
+        assert testInstance.query.aggs.to_dict()['aggs']['govDoc'] == {'terms': {'field': 'is_government_document', 'size': 2}}
+        assert testInstance.query.aggs.to_dict()['aggs']['govDoc_filter'] == {'filter': {'term': {'is_government_document': False}}}
+        assert testInstance.query.aggs.to_dict()['aggs']['editions'] == {'aggs': 
+        {'edition_filter_0': 
+            {'aggs': 
+                {'edition_filter_1': 
+                    {'aggs': 
+                        {'edition_filter_2': 
+                            {'aggs': 
+                                {'formats': 
+                                    {'aggs': {'editions_per': {'reverse_nested': {}}},
+                                    'terms': {'field': 'editions.formats', 'size': 10}},
+                                    'lang_parent': 
+                                        {'aggs': 
+                                            {'languages': 
+                                                {'aggs': 
+                                                    {'editions_per': {'reverse_nested': {}}},
+                                                'terms': {'field': 'editions.languages.language', 'size': 200}}},
+                                            'nested': {'path': 'editions.languages'}}},
+                                        'filter': {'exists': {'field': 'editions.formats'}}}},
+                                    'filter': {'terms': {'editions.formats': 'testFormat'}}}},
+                                'filter': {'range': {'editions.publication_date': 'testDate'}}}},
+                            'nested': {'path': 'editions'}}
```

Comment:
I'm torn on this (and for the record, don't change it).

I love the detail but it's tough to have this live in this file (just due to length). We have a `fixtures` directory that is used for a few tests but not widely. It might be good in the future to take blocks like this, store them as a JSON file and then load them as a test fixture that we can reference. Feels more maintainable to me.

Reply:
That makes sense. We can revisit this topic when we update and create future unit tests.